config/betterquesting/DefaultQuests.json

```
@@ -6029,12 +6381,12 @@
     "90:10": {
       "preRequisites:11": [
         91,
-        67
+        66
       ],
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Gallium and Arsenic do not exist as Ore!\n\n\nFor §dGallium§r, check the recipe for Tiny Dusts (open that handy material tab in JEI). You can obtain it from §acentrifuging impure dusts§r, either Bauxite or Sphalerite.\nYou can also §aelectrolyze Sphalerite§r, but the yield is really small. Furthermore, you will lose on smelting value the Sphalerite dust into Zinc.\n\n\nFor §dArsenic§r, you will have to mine and §acentrifuge Realgar§r. The ore is found in Galena veins.",
+          "desc:8": "Gallium and Arsenic do not exist as Ore!\n\n\nFor §dGallium§r, you may have to §aelectrolyze Sphalerite§r for a low chance of small dust. If this is the only recipe (including full, small and tiny dusts) to obtain Gallium in LV Age by the time you are playing this, yell at the devs for not hotfixing Ore Byproducts!\n\n\nFor §dArsenic§r, you will have to mine and §acentrifuge Realgar§r.",
```

```suggestion
          "desc:8": "Gallium and Arsenic do not exist as Ore!\n\n\nFor §dGallium§r, you may have to §aelectrolyze Sphalerite§r for a low chance of small dust.\n\n\nFor §dArsenic§r, you will have to mine and §acentrifuge Realgar§r.",
```

Ore byproducts have been fixed. You can obtaining by thermal centrifuging sphalerite, Centrifuging purified sphalerite, or persulfate bathing bauxite.